.github/workflows/ci.yml

```diff
@@ -23,24 +23,14 @@ jobs:
       run: bundle exec rspec
     - name: Run rubocop
       run: bundle exec rubocop --display-cop-names
-# This doesn't work on PRs from forks due to GITHUB_TOKEN, which makes
```

Comment:
I'd leave this... I think they've fixed this now and I don't wanna re-remember how to do this :)

Reply:
This wasn't fixed on the upstream repo and as I note in my summary, [it was ultimately archived as a result of it]( https://github.com/andrewmcodes/rubocop-linter-action). Or do you want to just see the attempt to do this?

Reply:
it wasn't an issue with upstream it was a limitation in how Actions which, which has been fixed.

Reply:
ugh, that's sad becasue it actually works now, but yeah kill it.